### PR TITLE
created migration files

### PR DESCRIPTION
**Pull request by samplesn08**
Description
What work was done?
• created new migration files for program-providers table and service-type-providers table
Why was this work done?
• In order to restructure the database to fit our desired schema
Type of change
• New feature (non-breaking change which adds functionality)
Change Status
• Completed, ready to review and merge
Checklist
• My code follows the style guidelines of this project
• I have performed a self-review of my own code
• My changes generate no new warnings
• There are no merge conflicts

### CODE REVIEW

**Comment by DomenicScarcella** [comment]
Nick, on lines 8-9 of program_providers, you have:

8        .references('id')
9        .inTable('programs')

I was following the dbdesigner schema when I made the programs table, and I called the id field "program_id" instead of "id" . . . will this cause a problem for program_providers trying to access that information from programs?

**Comment by samplesn08** [comment]
@DomenicScarcella 
You're right!!  My mistake.  I'll fix that

File: data/newMigrations/007-service_type_providers.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        .onUpdate('CASCADE')
+        .onDelete('CASCADE');
+        tbl
+        .string('provider_id')
+        .unsigned()
+        .notNullable()
+        .references('provider_id')
+        .inTable('providers')
+        .onUpdate('CASCADE')
+        .onDelete('CASCADE');
```
Comment (MatthewEllisTx) [inline data/newMigrations/007-service_type_providers.js:19]: I'm not sure we should be using CASCADE for onDelete or onUpdate. We should probably switch this to RESTRICT

File: data/newMigrations/005-program_providers.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('program_providers', (tbl) => {
+      tbl.increments('id').primary();
+      tbl
+      .integer('program_id', 128)
+      .unsigned()
+      .notNullable()
+      .references('program_id')
+      .inTable('programs')
+      .onUpdate('CASCADE')
```
Comment (MatthewEllisTx) [inline data/newMigrations/005-program_providers.js:10]: I don't think we should be using CASCADE for onDelete or onUpdate. We should probably switch this to RESTRICT

File: data/newMigrations/005-program_providers.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('program_providers', (tbl) => {
+      tbl.increments('id').primary();
+      tbl
+      .integer('program_id', 128)
```
Comment (MatthewEllisTx) [inline data/newMigrations/005-program_providers.js:5]: Please indent this.

tbl
    .integer('program_id', 128)
    ...
tbl...

**Review by MatthewEllisTx — COMMENTED** [review]
Overall looks solid to me, just do the changes I commented on please and ask me if you need any help

**Review by SpicyLunchbox — APPROVED** [review]
This meets merge specifications

**Review by MatthewEllisTx — APPROVED** [review]

**Pull request merged**